support/refc/mathFunctions.h

```diff
@@ -2,7 +2,6 @@
 
 #include "cBackend.h"
 #include <math.h>
-#include <gmp.h>
```

Comment:
Because it's included in cBackend.h. Actually there is another redundant one can be removed. I prefer to keep only one block of conditional includes, what's your idea?

Reply:
That makes sense. I assume these includes were leftovers from before it was also added to `cBackend.h`.